src/keys/slip10.rs

```diff
@@ -632,15 +628,23 @@ pub struct Bip44 {
 }
 
 impl Bip44 {
+    pub fn builder(self) -> Bip44Builder {
+        Bip44Builder(self)
```

Comment:
This is a very odd pattern. Usually a builder is used when you cannot default the base type.

Reply:
`Bip44` doesn't `impl Default`, the idea is to be able to easily convert an instance into a builder and modify/update it via the builder.

Reply:
Yes, but you take a value of `Bip44` in order to create the builder, which begs the question of how you create that without the builder. Catch-22.

Reply:
Bip44 value can be created from Bip44Builder or from [u32; 5].

Reply:
Sure, but if you already have a Bip44 then why do you need the builder?

Reply:
To construct a different bip44 chain. It can be done with clone and modifying a corresponding field, or it can be done with builder.

Reply:
That seems overly complex, why not just have accessor methods?

Reply:
Removed.